api/scripts/fill-skill-id-in-user-saved-tutorials.js

```diff
@@ -0,0 +1,114 @@
+const { knex } = require('../tests/test-helper');
+const UserSavedTutorial = require('../lib/domain/models/UserSavedTutorial');
+const UserSavedTutorialWithTutorial = require('../lib/domain/models/UserSavedTutorialWithTutorial');
+const knowledgeElementRepository = require('../lib/infrastructure/repositories/knowledge-element-repository');
+const tutorialDatasource = require('../lib/infrastructure/datasources/learning-content/tutorial-datasource');
+const skillDatasource = require('../lib/infrastructure/datasources/learning-content/skill-datasource');
+
+async function main() {
+  console.log('Starting filling skillId to user saved tutorials');
+
+  const userSavedTutorialsWithoutSkillId = await getAllUserSavedTutorialsWithoutSkillId();
+  const tutorials = await getAllTutorials();
+  const skills = await getAllSkills();
+
+  const tutorialsWithSkills = associateSkillsToTutorial(skills, tutorials);
+
+  for (const userSavedTutorialWithoutSkillId of userSavedTutorialsWithoutSkillId) {
+    const userSavedTutorial = associateTutorialToUserSavedTutorial(
+      userSavedTutorialWithoutSkillId,
+      tutorialsWithSkills
+    );
+    if (!userSavedTutorial.tutorial) {
+      return knex('user-saved-tutorials').delete().where({ id: userSavedTutorial.id });
```

Comment:
Le `return` dans une boucle `for` arrête l'exécution de la boucle. Je ne pense pas que c'est ce que l'on souhaite ici